### DISCUSSION TITLE
Redis Cluster Message Queue

### DISCUSSION THREAD

**Discussion by HanfordWu** [miguelgrinberg/python-socketio#849] [General]
Hi team,
As documentation shows, I am using aioredis client as a manager. I am going to move redis to redis cluster. But aioredis doesn't support  redis cluster mode. I don't have a second queuing software. I like to know what is your suggestion in this case? Thanks much!

**Comment by HanfordWu** [comment]
Another query is, aioredis did a big change on v2.* it's also going to support redis cluster mode in future release(pr is under review), but v2.* is not compatible with python-socketio. Do we have a plan to upgrade aioredis?

**Reply by HanfordWu** [reply]
Sorry, just found the change log of upgrading to aioredis2. No questions on this.

**Comment by miguelgrinberg** [comment]
Unfortunately I don't have anything to offer. The aioredis client manager is the only asyncio client that supports redis at this time. Sorry.

**Reply by HanfordWu** [reply]
Thanks anyway. I like to wait for the release of next aioredis and upgrading it in python-socketio. Before that I decided to keep a standalone redis for python-socketio message queue.
